### ISSUE TITLE
Windows 11 setup guidelines needed

### ISSUE THREAD

**Issue by vijayxd** [zju3dv/EasyMocap#431]
Can anyone please write step by step instructions to install and test on windows 11 

I followed existing guidelines but python scripts some modules are missing

**Comment by Nancyeleanor** [comment]
yes, badly needed, not able to install on windows. instructions are only provided for Linux.
